clang/lib/StaticAnalyzer/Core/BugReporter.cpp

```diff
@@ -287,6 +288,34 @@ class PathDiagnosticBuilder : public BugReporterContext {
   const PathSensitiveBugReport *getBugReport() const { return R; }
 };
 
+std::string timeTraceName(const BugReportEquivClass &EQ) {
+  if (!llvm::timeTraceProfilerEnabled())
+    return "";
+  const auto &bugReports = EQ.getReports();
```

Comment:
Nit: variables should start with upper case.

Reply:
fixed
b5792c2b135e [NFC] Uppercase variable names